docker-compose-auditor.yml

```diff
@@ -76,12 +76,15 @@ services:
       - SCALAR_DL_LEDGER_AUDITOR_ENABLED=true
 
   scalar-auditor:
-    image: ghcr.io/scalar-labs/scalar-auditor:3.9.1
-    # image: 709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor:3.9.1
+    image: ghcr.io/scalar-labs/scalardl-auditor-byol:3.9.1
     container_name: "scalardl-samples-scalar-auditor-1"
     volumes:
       - ./fixture/auditor.pem:/scalar/auditor.pem
       - ./fixture/auditor-key.pem:/scalar/auditor-key.pem
+      - ./fixture/auditor.properties.tmpl:/scalar/auditor/auditor.properties.tmpl
+      - ./fixture/trial-license-cert.pem:/scalar/license-cert.pem
+      # If you use commercial license key, you must use `commercial-license-cert.pem` instead of `trial-license-cert.pem`.
+      # - ./fixture/commercial-license-cert.pem:/scalar/license-cert.pem
```

Comment:
I set the certificate for a `trial license key` by default. However, users might use a `commercial license key` to run `scalardl-samples`. So, I added the configuration for the `commercial license key` as a commented out configuration.

I explain the use case of `commercial license` on the document side.